lucet-module-data/src/module_data.rs

```diff
@@ -23,6 +23,9 @@ pub struct ModuleData<'a> {
     globals_spec: Vec<GlobalSpec<'a>>,
     #[serde(borrow)]
     function_info: Vec<FunctionMetadata<'a>>,
+    #[serde(borrow)]
+    import_functions: Vec<ImportFunction<'a>>,
```

Comment:
I have these also as `Vec` because I expect the most common uses of these to be enumerating all imports or all exports of a module when doing setup, and probably not caring about import/export info after that point. If y'all can think of a point where we need to performantly look up if a given function is imported or exported, this probably isn't a great representation